DigitalOffice/Services/UserService/Broker/Requests/IInternalGetUserInfoRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Broker.Requests
+{
+    /// <summary>
+    /// DTO for getting user information through a message broker
+    /// </summary>
+    public interface IInternalGetUserInfoRequest
+    {
+        public Guid UserId { get; }
```

Comment:
public access modifier is redundant in Interfaces. In public interfaces all fields are public by default.

Reply:
deleted